### ISSUE TITLE
Lander coords not exactly stable over time, on other bodies.

### ISSUE THREAD

**Issue by Ezriilc** [Ezriilc/HyperEdit#41]
To reproduce:

- Launch any ship to pad or runway.
- Edit Orbit to Kerbin at any alt.
- Edit Orbit to Moho at any alt.
- Use Ship Lander to land at any location on Moho.
- Repeat these steps to land a 2nd ship, at the same coords.
- Observe that the same coords do NOT give the same location for the 2nd craft.

**Comment by fronbow** [comment]
I have a feeling I know what causes this; but as a general ask, what do you think should happen?
1. Should we attempt to land in exactly the same spot as a possible existing craft?
2. Or??

I imagine I'm creating a bit more work for myself if the answer is 1 as I'll need to check why the I,J,K,L keys don't work as they're supposed to for fine-tweaking landing.

**Comment by Ezriilc** [comment]
Number 1, for sure.  That, combined with the lateral adjustments available via JKLI, makes for a flexible, yet yet still controllable Landing.

**Comment by fronbow** [comment]
Okies, I'll look at that next!

**Comment by fronbow** [comment]
I think I've solved this in my latest build!
Though due to the nature of coords, the 2nd vessel doesn't spawn on top of the previous one, but a few metres away which feels a lot better.

**Comment by Ezriilc** [comment]
Can we increase the precision of our coords to make it perfect?  I think it's very important that HE does exactly as it's told.  The JKLI keys can be used if the user wishes to avoid an existing ship.  This is important for various reasons, like building bases.

**Comment by fronbow** [comment]
Yes, I removed the logic that was trying to add 50m to the coords so now it uses the exact same coords.
So using my moho test.
```
Two Stage Lander:
Lat: -0.000113031145345185
Lon: -171.65461843275

Orbiter 1A:
Lat: -0.000113031145345185
Lon: -171.65461843275
```
These are the numbers returned by using Current on the Lander.
KSPEngineer reports that the first one is at 171" 39' 16 W and the second craft is at 171" 39' 45 W which makes me think time factors in somehow as the precise values are exactly the same!
Maybe we need to change the advertisement to "can land you in the same major grid coords"?!!

**Comment by fronbow** [comment]
Did a few more tests recently and the coords do seem to change based on time, and also based on the time the craft launched. Obviously I need to do more testing to find a way of "fixing" this if it's possible!!

**Comment by Ezriilc** [comment]
It seems hard to believe that there is/are no API value(s) to hook onto.

In the worst case, perhaps can we count/track the change(s) over time to end up with the same thing the game does.
